src/validators/report-validator.js

```diff
@@ -0,0 +1,68 @@
+const  {check, param}  =  require ( 'express-validator' )
+const {StatusCodes} = require ('http-status-codes');
+const {validationAccountId} = require('../validators/discussion-validator');
+const {validateResult} = require('../helpers/response-result');
+
+const validationReport = [
+    check('reason')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
        .isString().withMessage("El campo debe ser una cadena")
```